oak/common/BUILD

```diff
@@ -53,6 +53,16 @@ cc_library(
     ],
 )
 
+cc_library(
```

Comment:
After the fact drive-by: is this target needed? Couldn't things that depend on `//oak/common:identity` just depend on `//oak_abi/proto:identity_cc_proto` instead?

Reply:
It is. Thanks. Fixed in #1705